app/controllers/pictures_controller.rb

```diff
@@ -17,7 +17,7 @@ def edit
   end
 
   def search
-      @pictures = if params[:search].blank?
+    @pictures = if params[:search].blank?
                     Picture.filter(params.slice(:author, :finish, :condition))
```

Comment:
indent like this ref: https://github.com/rubocop-hq/ruby-style-guide#source-code-layout at point **When assigning the result of a conditional expression to a variable, preserve the usual alignment of its branches.**